src/resources/ha-style.js

```diff
@@ -52,6 +52,7 @@ documentContainer.innerHTML = `<custom-style>
       --card-background-color: #ffffff;
       --primary-background-color: #fafafa;
       --secondary-background-color: #e5e5e5; /* behind the cards on state */
+      --edit-mode-header-background-color: #455a64; /* Edit Mode Changes Header Color */
```

Comment:
I don't like having names about how colors are going to be applied. We have primary, accent, maybe we should have `dark-background-color` and `dark-text-color` ?

Reply:
Or maybe we should define a theme for editing mode :D  and apply that to the app-header ?

Reply:
or
We could use `var(--dark-primary-color)` and fall back onto a set value

Reply:
`var(--dark-primary-color, #455a64);`

Reply:
We should change our values to HSL 😉

Reply:
We should actually. maybe someday Ill get bored enough to do that.